plugins/modules/apt_rpm.py

```diff
@@ -97,10 +129,33 @@ def query_package_provides(module, name):
 
 
 def update_package_db(module):
-    rc, out, err = module.run_command("%s update" % APT_PATH)
+    rc, out, err = module.run_command(["md5sum", "/var/cache/apt/pkgcache.bin"])
```

Comment:
Now you're adding a new dependency to the module: the `md5sum` executable. (Which might not even be present on FIPS enabled systems.)

Why not use `module.sha256('/var/cache/apt/pkgcache.bin')`?

Reply:
Good idea, thanks.